stubs/parsimonious/parsimonious/nodes.pyi

```diff
@@ -0,0 +1,40 @@
+from re import Match
+from typing import Any, Callable, Iterator, NoReturn, TypeVar
+from typing_extensions import Self
+
+from parsimonious.exceptions import UndefinedLabel as UndefinedLabel, VisitationError as VisitationError
+from parsimonious.expressions import Expression as Expression
+from parsimonious.grammar import Grammar as Grammar
+
+class Node:
+    expr: Expression
+    full_text: str
+    start: int
+    end: int
+    children: list[Node]
+    def __init__(self, expr: Expression, full_text: str, start: int, end: int, children: list[Node] | None = ...) -> None: ...
```

Comment:
Does it have to be a list? Would be preferable to write `Sequence` so we don't run into list invariance.

Reply:
Makes sense, will update the various visit methods similarly as well